iXBRLViewerPlugin/viewer/src/js/aspect.js

```diff
@@ -12,106 +12,110 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { QName } from './qname.js';
 import $ from 'jquery';
 import i18next from 'i18next';
+import { QName } from './qname.js';
 import { Period } from './period.js';
 import { Identifiers } from './identifiers.js';
 
-export function Aspect(a, v, report) {
-    this._aspect = a;
-    this._value = v;
-    this._report = report;
-}
-
-Aspect.prototype.name = function() {
-    return this._aspect;
-}
-
-Aspect.prototype.label = function() {
-    if (this._aspect == 'c') {
-        return "Concept";
+export class Aspect {
+    constructor(a, v, report) {
+        this._aspect = a;
+        this._value = v;
+        this._report = report;
     }
-    else if (this._aspect == 'p') {
-        return "Period";
-    }
-    else if (this._aspect == 'u') {
-        return "Unit";
-    }
-    else if (this._aspect == 'e') {
-        return "Entity";
-    }
-    else {
-        return this._report.getLabel(this._aspect);
+
+    name() {
+        return this._aspect;
     }
-}
 
-Aspect.prototype.value = function() {
-    return this._value;
-}
+    label() {
+        if (this._aspect == 'c') {
```

Comment:
preexisting: throughout this repo loose equality comparisons (`==` vs. `===`) are common. Is there a reason for that?

Reply:
Not a good one.  I've generally only used strict equality where it's necessary to avoid a false equality when comparing to `null` or `undefined`, but now that you raise it, I agree that it makes more sense to use strict equality anywhere that we're not actually relying on loose equality.

Reply:
Done in this file in 8f3cd6d.